.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       matrix:
         os:
           - ubuntu-22.04
-        features: ["", "--features pro", "--all-features"]
+        features: ["--features pro", "--all-features"]
```

Comment:
isn't this redundant as we have only one features?

Reply:
[363723b](https://github.com/geo-engine/geoengine/pull/994/commits/363723b528122a81d8dbb5f6d76f123979a36083)

we could do it like this...